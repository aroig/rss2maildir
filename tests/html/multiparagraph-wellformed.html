<p>This is a paragraph</p>
<p>This is a second paragraph</p>
<p>This is a third paragraph, we'll deliberately make this over seventy characters long so that it should wrap</p>
