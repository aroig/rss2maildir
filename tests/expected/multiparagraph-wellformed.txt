This is a paragraph

This is a second paragraph

This is a third paragraph, we'll deliberately make this over seventy
characters long so that it should wrap
